### PR TITLE
eslint installed

### PR DESCRIPTION
**Pull request by HubertKwiatkowski**

### CODE REVIEW

**Review by Sumick — CHANGES_REQUESTED** [review]
Almost there! We are missing script (specified in task requirements) in your `package.json`, so it will be possible to run linter on your source files.

**Comment by HubertKwiatkowski** [comment]
hm... chyba dodalem, ale czy z odpowiednimi parametrami, to juz nie jestem pewien - nie mialem stycznosci z zadnym lintem

File: package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "test": "jest",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "lint": "run eslint"
```
Comment (Sumick) [inline package.json:11]: **issue:** I think this script won't work. Try run `yarn lint` to validate it

**Review by Sumick — CHANGES_REQUESTED** [review]
This script won't work. Validate with `yarn lint`, to check how it behaves

**Review by Sumick — APPROVED** [review]
It works well now!

**Pull request merged**